spec/unit/failure_app_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+describe GDS::SSO::FailureApp, type: :request do
+  describe "#redirect" do
+    before do
+      Rails.application.routes.draw do
+        get "redirect", to: GDS::SSO::FailureApp.action(:redirect)
+      end
```

Comment:
Ah, I suspect doing this is trashing the actual routes table, which is why all the other tests are failing 😅  oops